client/src/modules/Schedule/components/TableView/TableView.tsx

```diff
@@ -122,39 +135,99 @@ export interface TableViewProps {
   statusFilter?: string;
 }
 
+export interface CombinedFilter {
```

Comment:
prefer use `type` in such cases

Reply:
Sure, could you please explain why so I could avoid the same problems in the future?
I thought it's more of a personal preference

Reply:
for consistency. I believe we mostly use `type` for things which are not supposed to be extended like interfaces. If it's an internal thing inside component, I would use `type` due to more available features. (e.g. you can't combine `string | { foo: string}` via interface)

Reply:
> for consistency. I believe we mostly use `type` for things which are not supposed to be extended like interfaces. If it's an internal thing inside component, I would use `type` due to more available features. (e.g. you can't combine `string | { foo: string}` via interface)

Got it, thank you